openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2236,6 +2215,118 @@ The following is a non-normative example of a Credential Request with Credential
 
 The value of the `credential` claim in the Credential Response MUST be a string that is an SD-JWT VC. Credentials of this format are already suitable for transfer and, therefore, they need not and MUST NOT be re-encoded.
 
+# Claims Description {#claims-description}
+
```

Comment:
Currently the _claims description object_ described in this section is being used for both:
- the metadata (information produced by the issuer).
- `authorization_details` entries (RAR) and credential request (information produced by the wallet), where the `value_type` and `display` keys cannot be used.

Instead of having a single object model, I believe it would be preferable to have two distinct object model sections, both sharing the path definition. IMO, the advantages are:
- We can grow each object model independently. For instance, in the future we can add policy information to the metadata model, without needing to explicitly forbid those new elements on RAR and credential requests.
- Have better descriptions for each key. For instance, even if the `mandatory` key appears on both, we could have distinct fine-tuned descriptions for each case. Currently the `mandatory` description text only applies to its use on the metadata and the meaning of `mandatory` on RAR and credential requests needs to be inferred.

Reply:
Hi Pedro, this is exactly what may suggestions do, please check

Reply:
Thanks, I have fixed this, please review the latest version.